core/pom.xml

```diff
@@ -270,7 +270,7 @@
     <dependency>
       <groupId>org.tachyonproject</groupId>
       <artifactId>tachyon-client</artifactId>
-      <version>0.8.1</version>
+      <version>0.8.2</version>
       <exclusions>
         <exclusion>
           <groupId>org.apache.hadoop</groupId>
```

Comment:
Ah, there was some slight confusion here: I somehow parsed "Distribution of Tachyon client module with shaded packages in jar" as "shades all dependencies", which isn't true, so there's not an issue here. My bad.